### DISCUSSION TITLE
How to upgrade versions?

### DISCUSSION THREAD

**Discussion by jsjoeio** [ish-app/ish#1428] [Q&A] [answered]
I know that 3.13 hasn't been tested but I'd like to try it out. I know that I need to edit `/etc/apk/repositories` but it looks like there is a version followed by a date.

Where can I find the date associated with the version?

Or, in simpler terms, what do I need to replace these lines with to upgrade to 3.13?

![ABFCF6B0-948C-416A-8E6F-21E54A8C3D09](https://user-images.githubusercontent.com/3806031/120116493-4c2b3d00-c13d-11eb-87dd-754325d47f64.jpeg)

**Comment by jsjoeio** [comment, marked as answer]
Okay I figured it out. 

## How to upgrade
1. `vi /etc/apk/repositories`
2. Replace the first line with `http://dl-cdn.alpinelinux.org/alpine/v3.13/main`
3. Replace the second line with `http://dl-cdn.alpinelinux.org/alpine/v3.13/community`
4. Save and exit `!wq`
5. `apk update`

**Reply by tbodt** [reply]
Ah, I had forgotten that the repositories file had been rewritten.

**Reply by jsjoeio** [reply]
Was this the correct way to upgrade, or is there a better way?

**Reply by tbodt** [reply]
Yeah this is the way.

**Reply by jsjoeio** [reply]
![this is the way](https://media.giphy.com/media/Ld77zD3fF3Run8olIt/giphy.gif)
